Source/WebCore/platform/graphics/gstreamer/WebKitWebSourceGStreamer.cpp

```diff
@@ -1010,6 +1010,15 @@ void CachedResourceStreamingClient::responseReceived(PlatformMediaResource&, con
 
     // length will be zero (unknown) if no Content-Length is provided or the response is compressed with Content-Encoding.
     uint64_t length = !response.httpHeaderFields().contains(HTTPHeaderName::ContentEncoding) ? response.expectedContentLength() : 0;
+
+    // But in some cases, Content-Encoding: chunked can specify the total length. Use it.
+    if (!length && response.httpHeaderFields().contains(HTTPHeaderName::TransferEncoding)
+        && (response.httpHeaderField(HTTPHeaderName::TransferEncoding) == String::fromUTF8("chunked"))) {
```

Comment:
`"chunked"_s`

Reply:
Check should be case insensitive, afaik

Reply:
Also, it looks like HTTP header values are case insensitive, so I should change it to lowercase before comparing.